lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,52 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        tunit = Unit('days since 0001-01-01 00:00:00', calendar='gregorian')
```

Comment:
@hdyson 
Just noticed this line (after a double-take) ... did you know that you can also do this `Unit('days since epoch')` ?

Which is kinda handy :wink: